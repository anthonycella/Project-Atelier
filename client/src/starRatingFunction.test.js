import renderer from 'react-test-renderer';
import QuarterStars from './starRatingFunction.jsx';

it('Quarter star function renders to the page', () => {
  const component = renderer.create(
    <QuarterStars rating={3.7} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
