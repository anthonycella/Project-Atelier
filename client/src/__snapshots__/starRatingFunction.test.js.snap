// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quarter star function renders to the page 1`] = `
<div
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "display": "inline-block",
        "height": "20px",
        "width": "18px",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "black",
          "display": "inline-block",
          "height": "20px",
          "position": "relative",
          "width": "18px",
        }
      }
    >
      <img
        alt="Star Icon"
        src="https://raw.githubusercontent.com/psfonseka/five-stars/master/dist/star.png"
        style={
          Object {
            "height": "20px",
            "width": "18px",
          }
        }
      />
    </div>
  </div>
  <div
    style={
      Object {
        "display": "inline-block",
        "height": "20px",
        "width": "18px",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "black",
          "display": "inline-block",
          "height": "20px",
          "position": "relative",
          "width": "18px",
        }
      }
    >
      <img
        alt="Star Icon"
        src="https://raw.githubusercontent.com/psfonseka/five-stars/master/dist/star.png"
        style={
          Object {
            "height": "20px",
            "width": "18px",
          }
        }
      />
    </div>
  </div>
  <div
    style={
      Object {
        "display": "inline-block",
        "height": "20px",
        "width": "18px",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "black",
          "display": "inline-block",
          "height": "20px",
          "position": "relative",
          "width": "18px",
        }
      }
    >
      <img
        alt="Star Icon"
        src="https://raw.githubusercontent.com/psfonseka/five-stars/master/dist/star.png"
        style={
          Object {
            "height": "20px",
            "width": "18px",
          }
        }
      />
    </div>
  </div>
  <div
    style={
      Object {
        "display": "inline-block",
        "height": "20px",
        "width": "18px",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "black",
          "display": "inline-block",
          "height": "20px",
          "position": "relative",
          "width": "12px",
        }
      }
    >
      <img
        alt="Star Icon"
        src="https://raw.githubusercontent.com/psfonseka/five-stars/master/dist/star.png"
        style={
          Object {
            "height": "20px",
            "width": "18px",
          }
        }
      />
    </div>
  </div>
  <div
    style={
      Object {
        "display": "inline-block",
        "height": "20px",
        "width": "18px",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "black",
          "display": "inline-block",
          "height": "20px",
          "position": "relative",
          "width": "0px",
        }
      }
    >
      <img
        alt="Star Icon"
        src="https://raw.githubusercontent.com/psfonseka/five-stars/master/dist/star.png"
        style={
          Object {
            "height": "20px",
            "width": "18px",
          }
        }
      />
    </div>
  </div>
</div>
`;
